src/components/TheNavbar.vue

```diff
@@ -22,23 +22,61 @@
       </ul>
     </div>
   </nav>
-
 </template>
 
 <script>
 export default {
   name: 'TheNavbar',
+  
+  data() {
+    return {
+      pushRight: true,
+      screenWidth: 0,
+    }
+  },
+  created() {
+        window.addEventListener('resize', this.handleResize);
+        this.handleResize();
+  },
+  unmounted() {
+        window.removeEventListener('resize', this.handleResize);
+  },
+  methods: {
+    handleResize() {
```

Comment:
@phann-phearoen 
I think you can handle responsive sizing by using v-col & v-row tags of Vuetify.

Below is a sample code.
Please try to replace it with Vuetify tags.
[(https://vuetifyjs.com/en/components/grids/#grid-system)](https://vuetifyjs.com/en/components/grids/#grid-system)

``` html
<v-container>
  <v-row>
   <v-col cols="12" sm="3"  class="ml-3">
    <div>Mamasuku</div>
   </v-col>
   <v-col cols="12" sm="6">
   </v-col>
   <v-col cols="12" sm="3">
   </v-col>
  </v-row>
</v-container>
```

If so, you don't need to create a function and other logics and the above code can control responsiveness by size of device too.

Only template part is what you need to write which makes your code much simpler.